Dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM folioci/alpine-jre-openjdk11:latest
+FROM folioci/alpine-jre-openjdk17:latest
+
+# Install latest patch versions of packages: https://pythonspeed.com/articles/security-updates-in-docker/
```

Comment:
Is this part of the documented steps for upgrading to JDK 17 or something additional included in this work?